vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -318,6 +319,7 @@
     <ClInclude Include="Views\Impl\SnapPointManagingContentControl.h" />
     <ClInclude Include="Views\IXamlRootView.h" />
     <ClInclude Include="Views\KeyboardEventHandler.h" />
+    <ClInclude Include="Views\PaperShadowNode.h" />
```

Comment:
We should make sure that we also update the nuspec for the Win32 package....
The file is copied in [Layout-Headers.ps1#L69](https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/Tfs/Layout-Headers.ps1#L69)
The file is then packaged in: [OfficeReact.Win32.nuspec#L51](https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/OfficeReact.Win32.nuspec#L51)

Reply:
Note: This might require changes in Office code as well...

Reply:
Thats a different shadownode file I believe.  (I know.. too many of them around)